src/main/java/cz/smarteon/loxone/app/Control.java

```diff
@@ -25,99 +25,58 @@
         @JsonSubTypes.Type(name = DigitalInfoControl.NAME, value = DigitalInfoControl.class),
         @JsonSubTypes.Type(name = AnalogInfoControl.NAME, value = AnalogInfoControl.class)
 })
+@Getter
 public abstract class Control {
 
-    @JsonProperty(value = "uuidAction", required = true)
-    protected LoxoneUuid uuid;
-
-    @JsonProperty(value = "name", required = true)
-    protected String name;
-
-    @JsonProperty(value = "isSecured")
-    protected boolean secured;
-
-    @JsonProperty(value = "type")
-    protected String type;
-
-    @JsonProperty(value = "room", required = true)
-    protected LoxoneUuid roomUuid;
-
-    @JsonProperty(value = "cat", required = true)
-    protected LoxoneUuid categoryUuid;
-
-    @JsonProperty(value = "defaultRating")
-    protected int rating;
-
-    @JsonProperty("states") @JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
-    protected Map<String, LoxoneUuids> states;
-
     /**
      * UUID of this control, should be unique
-     * @return control UUID
      */
-    @NotNull
```

Comment:
is it possible to configure Lombok to add this annotation to generated getter? If not it's a big loss, as this annotation makes the usage of the library on kotlin much safer

Reply:
thank you